scripts/install-test-dependencies.sh

```diff
@@ -2,5 +2,7 @@
 set -e
 
 pip3 install --no-cache-dir --ignore-installed --force-reinstall wheel pyyaml requests
-pip3 install --no-cache-dir --ignore-installed --force-reinstall fetchai-ledger-api==0.8.0a2
+# ???
+pip3 install  --no-cache-dir --ignore-installed --force-reinstall --editable git+https://github.com/wkaluza/ledger-api-py.git@pr_tx_status#egg=fetchai-ledger-api
```

Comment:
Reference to personal instead of upstream repo. If this is deliberate, document why and add a TODO for future replacement.

Reply:
This is deliberate and temporary - see explanation on Slack.